Makefile

```diff
@@ -4,7 +4,7 @@ PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
+CSSLIBDIR ?= $(PREFIX)/share/container-storage-setup
```

Comment:
just wanted to mention, %{_libdir} is /usr/lib64 on x86_64 systems in case you'd like to put things there. It also evaluates to /usr/lib on x86 systems. So we can just use %{_libdir} if you're not really keen on /usr/share